tests/test_modeling_t5.py

```diff
@@ -556,6 +556,32 @@ def model(self):
     def tokenizer(self):
         return T5Tokenizer.from_pretrained("t5-base")
 
+    @slow
+    def test_small_integration_test(self):
```

Comment:
This tests shows that the `EncoderDecoderAttn` layer should **not** use a `relative_position_bias`